Makefile

```diff
@@ -117,14 +117,18 @@ check-format:
 check-unused-dependencies:
 	mix deps.unlock --check-unused
 
+.PHONY: check-static-typing
+check-static-typing:
+	mix dialyzer
+
```

Comment:
@remi what do you think about this? I also include it in the `check` target.

Reply:
Yup! 👌